teuthology/task/install.py

```diff
@@ -776,10 +740,9 @@ def install(ctx, config):
 
     project = config.get('project', 'ceph')
 
-    global deb_packages
-    global rpm_packages
-    debs = deb_packages.get(project, [])
-    rpm = rpm_packages.get(project, [])
+    global PACKAGES
```

Comment:
I don't think so, but I was trying to make as little impact as possible with the change.